.travis.yml

```diff
@@ -1,13 +1,37 @@
 language: java
 sudo: required
+dist: trusty
+#group: edge
+
 script:
   - ./gradlew check
 cache:
   directories:
   - $HOME/.gradle
-jdk:
-  - oraclejdk8
-  - openjdk6
+
+before_install:
+  - export GRADLE_OPTS=-Xmx1024m
+
+matrix:
+  include:
+    - jdk: openjdk7
+    - jdk: oraclejdk8 # JDK 1.8.0_131-b11
+    # From https://github.com/reactive-streams/reactive-streams-jvm/pull/383
+    - jdk: oraclejdk9 # JDK 9-ea+174 provided by Travis will be replaced by 9+177 in "before_install"
+      before_install:
+        - cd ~
+        - wget http://download.java.net/java/jdk9/archive/177/binaries/jdk-9+177_linux-x64_bin.tar.gz
```

Comment:
FYI, looks like this has been deleted and needs to be updated whenever there is a newer JDK build.